### DISCUSSION TITLE
add a unlock recompense in guild daily

### DISCUSSION THREAD

**Discussion by ghosts-arch** [Crownicles/Crownicles#1736] [Suggestions fermées]
add a unlock recompense in guild daily (very rare in term of percentage) for unlock all the members of the guild actually in jail for free.

**Comment by KirIcare** [comment]
Hmm
Le truc c'est que c'est rare d'avoir plusieurs personnes d'une même guilde en prison si ce n'est une seule donc si en plus y'a un très faible pourcentage et qu'il faut que ça tombe pile le bon jour, je sais pas si ça va être utile à beaucoup de personne... Faut voir avec les têtes pensantes

**Comment by Nargacaura** [comment]
Vous voulez faire comme si le reste des membres plaidoyaient pour libérer leurs coéquipiers enfermés, mais... que se passerait-il si la guilde ne comporte que le membre enfermé, sachant qu'il est impossible de se libérer tout seul sauf en attendant 24h?
De plus, par quoi voulez-vous remplacer l'outcome de libération si vous tomberez sur cette éventuelle récompense et que personne n'est enfermé?

Pas sûr que ça soit une bonne idée, sur le coup... :/

**Reply by KirIcare** [reply]
A la limite faudrait qu'il y ai une condition (un mec de la guilde en prison) pour avoir une chance d'avoir ce guilddaily avec un message du style "les charges à votre encontre ont toutes étaient abandonnées !" mais comme a dit pagos je suis pas fan...

**Comment by BastLast** [comment]
Pk le truc est dans idée acceptées ?

**Reply by QQtin3** [reply]
> Pk le truc est dans idée acceptées ?

probablement créé là ?
Ou move par erreur par quelqu'un
